codegen/src/fetch_metadata.rs

```diff
@@ -83,15 +100,15 @@ pub async fn fetch_metadata_bytes(
     Ok(bytes)
 }
 
-/// Returns the raw, 0x prefixed metadata hex from the provided URL.
-pub async fn fetch_metadata_hex(
-    url: Url,
-    version: MetadataVersion,
-) -> Result<String, FetchMetadataError> {
-    let bytes = fetch_metadata_bytes(url, version).await?;
-    let hex_data = format!("0x{}", hex::encode(bytes));
-    Ok(hex_data)
-}
+// /// Returns the raw, 0x prefixed metadata hex from the provided URL.
+// pub async fn fetch_metadata_hex(
+//     url: Url,
+//     version: MetadataVersion,
+// ) -> Result<String, FetchMetadataError> {
+//     let bytes = fetch_metadata_bytes(url, version).await?;
+//     let hex_data = format!("0x{}", hex::encode(bytes));
+//     Ok(hex_data)
+// }
```

Comment:
This is also not needed I guess?

Reply:
Ooh good points; I forgot to remove the code I commented out!